circle.yml

```diff
@@ -33,13 +33,33 @@ jobs:
       - run:
           name: Clean gradle
           command: ./gradlew clean
+      - run:
+          name: Add preferences to gradle.properties file
+          command: |
+            rm -fr gradle.properties
+            echo "android.useAndroidX=true" >> gradle.properties
+            echo "android.enableJetifier=true" >> gradle.properties
+            echo "org.gradle.jvmargs=-Xmx4608M" >> gradle.properties
+            cat gradle.properties
       - run:
           name: Download Dependencies
           command: ./gradlew androidDependencies
       - save_cache:
           paths:
             - ~/.gradle
           key: jars-{{ checksum "build.gradle" }}-{{ checksum  "MapboxAndroidDemo/build.gradle" }}
+      - run:
+          name: Create google-services.json file
+          shell: /bin/bash -euo pipefail
+          command: |
+            echo "${GOOGLE_SERVICES_JSON_RELEASE}" > MapboxAndroidDemo/google-services.json
+      - run:
+          name: Add preferences to fabric.properties file
+          command: |
+            rm -fr MapboxAndroidDemo/fabric.properties
+            echo "apiKey=${FABRIC_API_KEY}" >> MapboxAndroidDemo/fabric.properties
+            echo "apiSecret=${FABRIC_API_SECRET}" >> MapboxAndroidDemo/fabric.properties
+            cat MapboxAndroidDemo/fabric.properties
```

Comment:
we shouldn't print secret env variables to the console

Reply:
Removed